input/pagecontent/reference.md

```diff
@@ -367,204 +447,238 @@ If the index is omitted, the first element of the array (`[0]`) is assumed.
+
+> **Note:** foo[x] choices are NOT addressed as foo[boolean], foo[Quantity], etc.
 
 **Example:**
 
-* Assign value[x] string value to "Hello World":
+* The path to the string data type of Observation.value[x]:
 
   ```
-  * valueString = "Hello World"
+  valueString
   ```
 
-* Assign the value[x] Reference value to an instance of the Patient resource:
+* The path to the Quantity within the SystolicBP component of the [Blood Pressure profile](https://www.hl7.org/fhir/bp.html):
 
   ```
-  * valueReference = Reference(EveAnyperson)
+  component[SystolicBP].valueQuantity
   ```
 
 #### Profiled Type Choice Paths
 
-In some cases, a type may be constrained to a set of possible profiles. To address a specific profile on that type, follow the path with square brackets (`[ ]`) containing the profile's `name`, `id`, or `url`.
+In some cases, a data type may be constrained to a set of possible profiles. To address a specific profile on that type, follow the path with square brackets (`[ ]`) containing the profile's `name`, `id`, or `url`.
 
 **Example:**
 
-* After constraining an address element to be either a USAddress or a CanadianAddress, bind the address.state properties to value sets of US states and Canadian provinces:
+* After constraining Patient.contact.address to be either a USAddress or a CanadianAddress (assuming these are profiles on Address), the paths to the respective `state` elements in the first contact:
 
   ```
-  * address only USAddress or CanadianAddress
-  * address[USAddress].state from https://www.usps.com
-  * address[CanadianAddress].state from http://ehealthontario.ca/fhir/CodeSystem/province-state-codes
+  contact[0].address[USAddress].state
+
+  contact[0].address[CanadianAddress].state
   ```
 
 #### Extension Paths
 
-FHIR provides for user customization through extensions. Extensions are added to resources by populating pre-existing arrays designated for this purpose. Most resources have extension and a modifierExtension arrays at the top level, inherited from [DomainResource](http://hl7.org/fhir/R4/domainresource.html) (exceptions include Bundle, Parameters, and Binary). Every element of a resource also has an extension array, inherited from [Element](https://www.hl7.org/fhir/element.html). These arrays contain elements of [data type Extension](https://www.hl7.org/fhir/extensibility.html#extension). Particular types of extensions are created by profiling Extension.
+Extension arrays are found at the root level of every resource, nested inside every element, and recursively inside each extension. Extensions are elements in these arrays. When an extension is added to an extension array, a name (technically, a slice name) is assigned. Extensions can be identified by that slice name, or the extension's URL.
```

Comment:
I'm not thrilled with this because we don't define complex extensions. An extension becomes complex when an extension array is populated. Declining to change.